service/src/main/java/com/myhome/controllers/mapper/SchedulePaymentApiMapper.java

```diff
@@ -125,7 +126,7 @@ default EnrichedSchedulePaymentRequest enrichSchedulePaymentRequest(
         .collect(Collectors.toSet());
     return new EnrichedSchedulePaymentRequest(request.getType(),
         request.getDescription(),
-        request.isRecurring(),
+        request.getRecurring(),
```

Comment:
I think we can add `"booleanGetterPrefix" : "is"` to the `additionalProperties` in `api/build.gradle` and the getters will be with `is` prefix without the changes :)

Reply:
Yes, I think that would be better. May I do that change @mslowiak ?

Reply:
@edwin9870 Yes, please :)